Libraries/react-native/react-native.js

```diff
@@ -172,6 +173,11 @@ const ReactNative = {
       return require('update');
     },
   },
+
+  // better error messages when accessing React APIs on ReactNative
+  get render() { throwOnWrongReactAPI('render'); },
```

Comment:
Is it possible to make these `__DEV__` only? (preferably the whole definition, otherwise just the method contents).